pkg/authorization/generated/informers/externalversions/authorization/v1/interface.go

```diff
@@ -1,83 +0,0 @@
-// This file was automatically generated by informer-gen
-
-package v1
-
-import (
-	internalinterfaces "github.com/openshift/origin/pkg/authorization/generated/informers/externalversions/internalinterfaces"
-)
-
-// Interface provides access to all the informers in this group version.
-type Interface interface {
-	// ClusterPolicies returns a ClusterPolicyInformer.
-	ClusterPolicies() ClusterPolicyInformer
-	// ClusterPolicyBindings returns a ClusterPolicyBindingInformer.
-	ClusterPolicyBindings() ClusterPolicyBindingInformer
-	// ClusterRoles returns a ClusterRoleInformer.
-	ClusterRoles() ClusterRoleInformer
-	// ClusterRoleBindings returns a ClusterRoleBindingInformer.
-	ClusterRoleBindings() ClusterRoleBindingInformer
-	// Policies returns a PolicyInformer.
-	Policies() PolicyInformer
-	// PolicyBindings returns a PolicyBindingInformer.
-	PolicyBindings() PolicyBindingInformer
-	// Roles returns a RoleInformer.
-	Roles() RoleInformer
-	// RoleBindings returns a RoleBindingInformer.
-	RoleBindings() RoleBindingInformer
-	// RoleBindingRestrictions returns a RoleBindingRestrictionInformer.
-	RoleBindingRestrictions() RoleBindingRestrictionInformer
```

Comment:
This is the only resource that could ever need an informer in the future.

Reply:
I was wondering why you were deleting that one too indeed.